pkg/crc/preflight/preflight_checks_linux.go

```diff
@@ -392,6 +397,65 @@ func fixLibvirtCrcNetworkActive() (bool, error) {
 	return true, nil
 }
 
+func checkLibvirtCrcBridgePermissions() (bool, error) {
+	logging.Debug("Checking if 'crc' bridge has appropriate permissions setup")
+	configPath := ""
+	for _, path := range qemuBridgeConfigPaths {
+		logging.Debug(fmt.Sprintf("Trying %s..", path))
+		_, err := os.Stat(path)
+		if err != nil {
+			logging.Debug(fmt.Sprintf("Failed to open %s: %s", path, err))
+		} else {
+			configPath = path
+
+			break
+		}
+	}
+	if configPath == "" {
+		return false, fmt.Errorf("Failed to find Qemu bridge configuration file")
+	}
+
+	config, err := ioutil.ReadFile(filepath.Clean(configPath))
+	if err != nil {
+		return false, fmt.Errorf("Failed to read %s: %v", configPath, err)
+	}
+	if match, _ := regexp.MatchString(qemuBridgeConfig, string(config)); !match {
```

Comment:
Is `MatchString` automatically anchored to the end of the line? Or could we get false positives for `allow crcfoo`?

Reply:
it is not... and matches a pattern at any place... so `not allow crc` would match or `//allow crc`.

Reply:
As you can see in the newly pushed code, I went for `^allow crc$` but for some reason that doesn't match, and neither does `^allow crc`.

Reply:
ok, with @cfergeau 's help this is resolved now. It's not a perfect solution but i think it'd do for now. PTALA.